getting-started/vs-integration/options.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Options
```

Comment:
I suggest changing the title so that you apply the same action-oriented style used by the rest of the titles in this docs section.
```suggestion
title: Configuring the VS Extensions
```
![image](https://github.com/telerik/blazor-docs/assets/60105689/b48d3cf9-daa6-4088-97bf-15a991bdf7c9)